proto/abis/massa/abi/v1/abi.proto

```diff
@@ -392,3 +399,107 @@ message DivRemNativeAmountResult {
   // Remainder of amount and divisor
   massa.model.v1.NativeAmount remainder = 2;
 }
+
+// Comparison result
+message ComparisonResult {
```

Comment:
@aoudiamoncef 
Do you think I should put this one in commons?
I feel like it should be

Reply:
Yes I think so too, a bit like https://doc.rust-lang.org/std/cmp/enum.Ordering.html

Reply:
let's agree on the path before I move it
I propose 
`proto/commons/massa/shared/v1/comparison.proto`

it follow the scheme of others
`proto/abis/massa/abi/v1/abi.proto`
`proto/abis/massa/abi/v1/abi.proto`
`proto/commons/massa/model/v1/address.proto`...